commands/install.go

```diff
@@ -73,6 +79,7 @@ func initInstallFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(install.FlagOperatorMongoDB, true, "Install MongoDB operator")
 	cmd.Flags().Bool(install.FlagOperatorPostgresql, true, "Install PostgreSQL operator")
 	cmd.Flags().Bool(install.FlagOperatorXtraDBCluster, true, "Install XtraDB Cluster operator")
+	cmd.Flags().BoolP("logs", "l", false, "If set, logs are printed during the installation process")
```

Comment:
What do you think about re-purposing the `-v` flag for this? I think it's more natural for CLI users to use `-v` for increasing the logging level.
We can for example have `-v` for adding the logs with the `Info` level and `-vv` for `Debug` logs. Or even just use the `Debug` level by default and have just `-v`. WDYT?

Reply:
Yeah, that's a good idea.. updated 👍🏻